app/src/main/java/com/beemdevelopment/aegis/ui/MainActivity.java

```diff
@@ -94,6 +96,34 @@ public class MainActivity extends AegisActivity implements EntryListView.Listene
 
     private ActionMode.Callback _actionModeCallbacks = new ActionModeCallbacks();
 
+    ActivityResultLauncher<Intent> scanLauncher = registerForActivityResult(new ActivityResultContracts.StartActivityForResult(),
```

Comment:
Now that we're using the new API, the ``CODE_`` request codes above are no longer used and can be removed.